### ISSUE TITLE
Live demo?

### ISSUE THREAD

**Issue by impressivewebs** [mrdoob/texgen.js#27]
The main site doesn't seem to be a demo, but just displays a PNG of some samples. Can't you add a live example on that page, to demonstrate what it doesn? You might also want to explain that the rendered result is a canvas element (which I'm assuming because of the method name in your code example). Thanks!

**Comment by mrdoob** [comment]
Yes. I want to make the site nicer. Hoping to get some time for that soon!

**Comment by jeromeetienne** [comment]
@mrdoob if you got directions, im definitly willing to help.

**Comment by zoalst** [comment]
I made a [demo](http://goo.gl/hcHvPk) yesterday so I could learn how this works. The code is [here](https://github.com/zacharystenger/texgen.js/blob/master/examples/demo.html). I am also willing to help out.

**Comment by mrdoob** [comment]
I pushed on dd4e96ca3f410e2bdcf0719e9118f8a4f1d2f7fe the editor I'm working on.
